### PR TITLE
task done, all tests passed

### PR DESCRIPTION
**Pull request by Manankin**
Done !

### CODE REVIEW

File: src/formatDate.js [hunk 1]
```diff
@@ -51,6 +51,63 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const NEW_DIVIDER = toFormat[3];
```
Comment (vSainiuk) [inline src/formatDate.js:54]: I think it is better to name the variable like 'separator' for better understanding)

**Review by vSainiuk — APPROVED** [review]
I like it. Well done!

File: src/formatDate.js [hunk 2]
```diff
@@ -51,6 +51,63 @@
...
+        if (index < 0) {
+          index = fromFormat.indexOf('YYYY');
+        }
+
+        number = dateCopy[index];
+
+        if (number <= 99) {
+          number = number >= 30 ? '19' + number : '20' + number;
+        }
+        result.push(number);
```
Comment (DanilWeda) [inline src/formatDate.js:91]: Use a function for all case results.

**Review by DanilWeda — CHANGES_REQUESTED** [review]
Thanks for a good job! 
Check comments for improvement!

File: src/formatDate.js [hunk 3]
```diff
@@ -51,6 +51,75 @@
...
+    let index = fromFormat.indexOf('YY');
+
+    if (index < 0) {
+      index = fromFormat.indexOf('YYYY');
+    }
+
+    let number = dateCopy[index];
+
+    if (number <= 99) {
+      number = number >= 30 ? '19' + number : '20' + number;
```
Comment (olya-shyian) [inline src/formatDate.js:117]: 99, 30 , 19, 20 what do these magic numbers mean?
could you use variables for them?

**Review by olya-shyian — CHANGES_REQUESTED** [review]
Great! You did a nice job!)

**Review by olya-shyian — APPROVED** [review]
Excellent!🤩

**Pull request closed without merging**